libs/agentc_cli/agentc_cli/cmds/publish.py

```diff
@@ -136,7 +136,7 @@ def cmd_publish(
         progress_bar = tqdm.tqdm(catalog_desc.items)
         for item in progress_bar:
             try:
-                key = item.identifier
+                key = item.identifier + "_" + metadata["version"]["identifier"]
```

Comment:
in a case where there is no change to a tool and user simply commits and then indexes and publishes the catalog, the identifier being initial hash + current catalog hash will create a new document - this will bloat up the storage over time

can we instead add the catalog identifier in another field?

Reply:
following up on this bit -- from the call earlier today, this should be fine (mainly to make search a bit easier + minimizing the changes for reInvent release)

in a followup change, we can work towards minimizing the storage by adding "dummy" docs that point to the unchanged file (and resolve these with a JOIN + a UNION) 

also, gave a look at the status command and the behavior is unchanged (the catalog_identifier is used with DISTINCT, and returns the correct number of records for --status-db calls)